mutation-testing/output-pr-mutants/action.yml

```diff
@@ -0,0 +1,166 @@
+              echo "</details>" >> "$GITHUB_STEP_SUMMARY"
+              echo "" >> "$GITHUB_STEP_SUMMARY"
+            elif [[ "$section_title" == "Unviable:" ]]; then
+              echo "<details>" >> "$GITHUB_STEP_SUMMARY"
+              echo "<summary>What are unviable mutants?</summary>" >> "$GITHUB_STEP_SUMMARY"
+              echo "<br>" >> "$GITHUB_STEP_SUMMARY"
+              echo "The attempted mutation doesn't compile. This is inconclusive about test coverage and no action is needed, unless you wish to test the specific function, in which case you may wish to add a 'Default::default()' implementation for the specific return type." >> "$GITHUB_STEP_SUMMARY"
+              echo "</details>" >> "$GITHUB_STEP_SUMMARY"
+              echo "" >> "$GITHUB_STEP_SUMMARY"
+            fi
+
+            if [[ "$section_title" != "" ]]; then
+              awk -F':' '{printf "- [ ] " "[" $0 "]"; file_path=$1; line=$2; $1=""; $2=""; printf "(" "'"$server_url"'/'"$organisation"'/'"$repository"'/blob/'"$commit"'/" file_path "#L" line-1 ")\n\n"}' "$file_name" >> "$GITHUB_STEP_SUMMARY"
+            else
+              awk -F':' '{printf "- [x] " "[" $0 "]"; file_path=$1; line=$2; $1=""; $2=""; printf "(" "'"$server_url"'/'"$organisation"'/'"$repository"'/blob/'"$commit"'/" file_path "#L" line-1 ")\n\n"}' "$file_name" >> "$GITHUB_STEP_SUMMARY"
+            fi
+
+            if [[ "$section_title" == "Missed:" ]]; then
+              echo "### To resolve this issue, consider one of the following options:" >> "$GITHUB_STEP_SUMMARY"
+              echo "- Modify or add tests including this function." >> "$GITHUB_STEP_SUMMARY"
+              echo "- If you are absolutely certain that this function should not undergo mutation testing, add '#[mutants::skip]' or '#[cfg_attr(test, mutants::skip)]' function header to skip it." >> "$GITHUB_STEP_SUMMARY"
+            elif [[ "$section_title" == "Timeout:" ]]; then
+              echo "### To resolve this issue, consider one of the following options:" >> "$GITHUB_STEP_SUMMARY"
+              echo "- Modify the tests that include this funcion." >> "$GITHUB_STEP_SUMMARY"
+              echo "- Add '#[mutants::skip]' or '#[cfg_attr(test, mutants::skip)]' function header to skip it." >> "$GITHUB_STEP_SUMMARY"
+            elif [[ "$section_title" == "Unviable:" ]]; then
+              echo "### To resolve this issue, consider one of the following options:" >> "$GITHUB_STEP_SUMMARY"
+              echo "- Create 'Default::default()' implementation for the specific structure." >> "$GITHUB_STEP_SUMMARY"
+              echo "- Add '#[mutants::skip]' or '#[cfg_attr(test, mutants::skip)]' function header to skip it." >> "$GITHUB_STEP_SUMMARY"
+            fi
+
+            echo >> "$GITHUB_STEP_SUMMARY"
+          fi
```

Comment:
We don't need an exit or an echo here because any file out of the 4 tested (missed, timeout, unviable, and caught) can miss, as long as there's at least 1 existing (in the case all are missing, the build fails earlier in the `Check Packages and Shards` job).